ffi/diplomat/tests/missing_apis.txt

```diff
@@ -22,6 +22,20 @@ icu::collections::codepointinvliststringlist::CodePointInversionListAndStringLis
 icu::collections::codepointinvliststringlist::CodePointInversionListAndStringList::try_from#FnInStruct
 icu::collections::codepointinvliststringlist::CodePointInversionListAndStringListError#Enum
 icu::collections::codepointinvliststringlist::Error#Enum
+icu::compactdecimal::CompactDecimalError#Enum
```

Comment:
nit: can you add the crate to the list of experimental crates in the coverage test